src/pages/homepage/ExploreService.jsx

```diff
@@ -1,14 +1,14 @@
 import { gsap } from 'gsap';
 import { useGSAP } from '@gsap/react';
-import { ourServiceImage1, ourServiceImage2, ourServiceImage3, rightArrowIcon } from '../../assets';
 import { useRef } from 'react';
+import { ourServiceImage1, ourServiceImage2, ourServiceImage3, rightArrowIcon } from '../../assets';
 
 const ExploreService = () => {
     const containerRef = useRef(null);
+
     useGSAP(() => {
         const container = containerRef.current;
-        const images = container.querySelectorAll('.image-container');
-        gsap.from(images, {
+        gsap.from(container.querySelectorAll('.image-container'), {
```

Comment:
this is same as previous

Reply:
sir iye mene isliye use kiya hai kioki iye animation 3 jagah use krna tha image-container class use krke

Reply:
isliye iye maping krke use krna pda

Reply:
or koi another method se nhi ho rha sir isliye aisa kiya